Resources/Storage/StorageFactory.py

```diff
@@ -116,22 +117,22 @@ def getStorages( self, storageName, pluginList = None, hideExceptions = False ):
       seConfigPath = SE_CONFIG_PATH
 
     # Get the options defined in the CS for this storage
-    res = self._getConfigStorageOptions(storageName, derivedStorageName = derivedStorageName, 
+    res = self._getConfigStorageOptions(storageName, derivedStorageName=derivedStorageName,
                                         seConfigPath=seConfigPath)
     if not res['OK']:
       # This is for the backward compatibility and to invite developer to move their BaseSE in the correct section
-      gLogger.warn("Deprecated configuration, you can ignore the error message above."\
-                     " Please move the baseSE in the correct section: ", SE_BASE_CONFIG_PATH)
+      gLogger.warn("Deprecated configuration, you can ignore the error message above."
```

Comment:
About that ! Anyone has an explanation for the following behavior ?
```

In [13]: type('a' 'b')
Out[13]: str

In [14]: type(1 2)
  File "<ipython-input-14-46064c022beb>", line 1
    type(1 2)
           ^
SyntaxError: invalid syntax

```

Reply:
AFAIU `'a' 'b'` is equivalent to `'a'+'b'` while this is not possible with an `int`
My 2 cents

Reply:
implicit string concatenation, cf. http://legacy.python.org/dev/peps/pep-3126/

Reply:
Thanks !